src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(RandomColor.values().length);
```

Comment:
Create class variable `Random`and initialize it instead of creating every time new instance of this class. Fix it in the whole code.